lib/validator/v3/validator.go

```diff
@@ -278,6 +279,12 @@ func validateMAC(fl validator.FieldLevel) bool {
 	return true
 }
 
+func validateIptablesBackend(fl validator.FieldLevel) bool {
+	s := fl.Field().String()
+	log.Debugf("Validate Iptables Backend: %s", s)
+	return s == "" || s == "nft" || s == "legacy"
```

Comment:
For k8s resources the convention is that named values start with an uppercase.

e.g.,

`iptablesBackend: Legacy` or `iptablesBackend: NFTables` 

We probably want to use a type with some constants for this as well.

```
type IPTablesBackend string
```